Tools/CMake/JASP.cmake

```diff
@@ -92,19 +92,25 @@ message(CHECK_START "Checking for CRYPT_KEY")
 set(CRYPT_KEY
     ""
     CACHE STRING "")
+
 if(CRYPT_KEY STREQUAL "")
-  set(CRYPT_KEY $ENV{ENVIRONMENT_CRYPTKEY})
+	# lets see if the user set something in the environment
+  set(CRYPT_KEY $ENV{CRYPTKEY})
 
   if(CRYPT_KEY STREQUAL "")
-    message(CHECK_FAIL "not found.")
-    message(WARNING "CRYPT_KEY is not found.")
+	  message(CHECK_PASS "found default.")
+	  
+    message(WARNING "CRYPT_KEY is not found, using default replace if this is to go out to users!")
+	set(CRYPT_KEY "0x0c2ad4a4acb9f023")
+	
   else()
-    message(CHECK_PASS "found.")
+    message(CHECK_PASS "found in environment.")
     message(STATUS "  ${CRYPT_KEY}")
```

Comment:
I'd put this after the `if` that it prints the key either way.

Reply:
Well, I thought it might be useful to get feedback on where it came from?
You know, in case something is set up wrong?

On the other hand, it should also be obvious from the key itself